2009-12-02  Mike Frysinger <vapier@gentoo.org>

	* sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c: Fix typo in
	include timer_gettime.c -> timer_settime.c.

---
 ports/sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/ports/sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c
+++ b/ports/sysdeps/unix/sysv/linux/alpha/nptl/timer_settime.c
@@ -1 +1 @@
-#include <nptl/sysdeps/unix/sysv/linux/x86_64/timer_gettime.c>
+#include <nptl/sysdeps/unix/sysv/linux/x86_64/timer_settime.c>
